### ISSUE TITLE
Possibility to pass checkForExisting props to react-script-hook 

### ISSUE THREAD

**Issue by tmocellin** [hupe1980/react-recaptcha-hook#13]
Hi, thanks for this hooks.
I'm using it in a modal form and I'm asking if it would make sense to add the possibility to add a the prop `checkForExisting` for the useScript hook to avoid the multiple script tag each I open the modal. 
Something like that : 
 ```ts
export function useRecaptcha({
  sitekey,
  hideDefaultBadge = false,
 checkForExisting= false,
}: RecaptchaProps): (action: string) => Promise<string> {
  const [recaptcha, setRecaptcha] = React.useState<Recaptcha | undefined>()
  React.useEffect(() => {
    if (isBrowser && hideDefaultBadge) {
      injectStyle('.grecaptcha-badge { visibility: hidden; }')
    }
  }, [hideDefaultBadge])

  useScript({
    src: `https://www.google.com/recaptcha/api.js?render=${sitekey}`,
    onload: () =>
      (window as any).grecaptcha.ready(() => {
        console.log('recaptcha loaded')
        setRecaptcha((window as any).grecaptcha)
      }),
    checkForExisting 
 })
...
 ```

I can make a PR if you want.

**Comment by hupe1980** [comment]
@tmocellin  Why do you want to make it configurable? I think we can always set checkForExisting = true
You are welcome to create a PR!

**Comment by tmocellin** [comment]
@hupe1980 In my case each times I open / close my Auth modal a new script tag is added. I just want to  pass the checkForExisting prop to useScript inside the useRecaptcha code.

**Comment by hupe1980** [comment]
Why not as standard behavior?
```typescript
useScript({
   src: `https://www.google.com/recaptcha/api.js?render=${sitekey}`,
   onload: () =>
     (window as any).grecaptcha.ready(() => {
       console.log('recaptcha loaded')
       setRecaptcha((window as any).grecaptcha)
     }),
   checkForExisting : true,
})```

**Comment by tmocellin** [comment]
I think I'am not clear in my explanation I made a PR. #14

**Comment by hupe1980** [comment]
closed with #14
